packages/medusa/src/strategies/tax-calculation.ts

```diff
@@ -96,10 +96,25 @@ class TaxCalculationStrategy implements ITaxCalculationStrategy {
   ): number {
     let taxTotal = 0
     for (const sm of shipping_methods) {
-      const amount = sm.price
       const lineRates = taxLines.filter((tl) => tl.shipping_method_id === sm.id)
       for (const lineRate of lineRates) {
-        taxTotal += Math.round(amount * (lineRate.rate / 100))
+        if (
```

Comment:
comment: same as the previous one

Reply:
This FF is ok here, before calling calculatePriceTaxAmount where the FF will be removed.
The if/else block is to avoid `includesTax: FF && sm.includes_tax ?? false`, making it easier when we clean up the FF.

Reply:
I don’t think the` ?? false` is necessary in your example. It would be` !!(FF && sm.includes_tax)` wdyt? The thing is that if you store it before the loop, it would avoid having the re execute it for each loop knowing that the value does not change. 

I am fine with both but i would like your opinion.

Reply:
True. but most places where we have `?? false` I would also remove them. I'm fine with checking `if(undefined)` instead of forcing `if(false)`. 
But moving the FF outside the loop is ok to me.

Reply:
Ahah this is not what i meant. I ll write the example tomorrow or later when i get to my computer ☺️